samples/token-server/README.md

```diff
@@ -0,0 +1,93 @@
+
+* Limit token using CORS
+* Throttle token requests
+* Log and audit every requests
+* Rotate secret base on a schedule
+
+Your Direct Line secret is very important. Please handle them carefully and treat them as a physical key to all your user data. If the key is exposed, you must regenerate them immediately.
+
+# How to use
+
+1. Create a new GitHub repository
+1. Enable Travis CI for your repository
+1. Create a new Azure Container Registry
+   1. Enable admin user and write down the username and password
+1. Add environment variables to Travis CI settings
+   * `ACR_NAME` is the name of Azure Container Registry, will be appended with `.azurecr.io`
+   * `DOCKER_IMAGE_NAME` is the name of the image
+   * `DOCKER_PASSWORD` is the password of Azure Container Registry
+   * `DOCKER_USERNAME` is the username of Azure Container Registry
+1. Copy and commit source code from this directory to root of your GitHub repository
+   * This will kick off a build on Travis CI
+   * Verify the build is successful on Travis CI
+   * Verify the Docker image is pushed to Azure Container Registry
+1. Create a new Azure Web App for Containers
+   * Click "Configure container"
+      * Select "Single Container"
+      * "Image source" set to "Azure Container Registry"
+      * "Registry" set to your Azure Container Registry
+      * "Image" set to your Docker image name
+      * "Tag" set to "latest"
+      * "Startup File" leaves as blank
+      * Click "Apply" and start creating the Azure Web App
+   * Verify your new Azure Web App has deployed correctly
+1. Setup additional settings in your Azure Web App
+   * Set up your Direct Line secret
+      * Go to "Application settings"
+        * Under "Application settings" section, add new settings
+            * `DIRECT_LINE_SECRET` to your Direct Line secret
+            * (Optional) `BING_SPEECH_SUBSCRIPTION_KEY` to your Bing Speech subscription key, if any
+        * Click "Save"
+   * Enable continuous deployment in your Azure Web App
+      * Go to "Container settings"
+         * Set "Continuous Deployment" to "On"
+         * Click "Save"
+      * Verify by going to Azure Container Registry, check if a new Webhook for the Azure Web App
+   * (Optional) Improve application performance
+      * Go to "Application settings"
+         * Set "Always On" to "On"
+            * The web server will run indefinitely even there are no requests queued, it will be restarted (via heartbeat) if the process is down
+         * Set "ARR Affinitiy" to "Off"
+            * This web server is stateless and does not requires sticky session
+         * Click "Save"
+1. Verify if the Azure Web App is deployed correctly
+   * In Command Prompt or terminal, run `curl -d "" https://YOUR_AZURE_WEB_APP.azurewebsites.net/directline/token`
```

Comment:
Instead of an empty string, should this be something like "Authorization: token <DIRECT_LINE_SECRET>"? An empty string isn't indicative to the users what should be going in here, unless it should truly be empty.